tests/should_fail/alloc_max_size.coma

```diff
@@ -1,28 +1,31 @@
-module M_alloc_max_size__main
+module M_main
   use creusot.int.UInt64
   use creusot.slice.Slice64
   use seq.Seq
   use mach.int.Int
   use creusot.prelude.Any
   
-  predicate inv [@inline:trivial] (_1: Seq.seq UInt64.t) = true
+  predicate inv_Seq_usize [@inline:trivial] (_1: Seq.seq UInt64.t) = true
   
-  meta "rewrite_def" predicate inv
+  meta "rewrite_def" predicate inv_Seq_usize
   
-  predicate invariant' (self: Slice64.array UInt64.t) =
-    inv (Slice64.view self) /\ Seq.length (Slice64.view self) = UInt64.t'int (18446744073709551615: UInt64.t)
+  predicate invariant_array_usize_18446744073709551615 (self: Slice64.array UInt64.t) =
```

Comment:
Here.

Reply:
That's the length of the array! (BTW I think this test is broken, it should fail but in reality it would fail at codegen time, and our tests stop before that.)